src/book/02-system/01-foundations/03-continuous-integration.mdx

```diff
@@ -49,6 +49,28 @@ To view the logs for a build, scroll to the build checks at the bottom of the PR
 
 ![Screenshot of the build checks section of the PR page, with arrow pointing to primary link](./images/ci-build-logs.png 'Link to the build details with logs')
 
+## Pacman Package Cache
+
+When Buildkite receives a webhook to build and test a PR, the Buildkite agents pulls down the code and starts building a Docker image based off the Dockerfile in that branch.
+
+In the process of building our docker image, serveral [pacman](https://wiki.archlinux.org/index.php/pacman) packages are downloaded from public mirrors (if that specific [layer](https://docs.docker.com/storage/storagedriver/#images-and-layers) itself is not cached). Because these packages are downloaded numerous times, across multiple PR's, caching them locally can speed up build times and save network bandwidth.
+
+It is also possible that the [signature](https://wiki.archlinux.org/index.php/Pacman/Package_signing) on a package that is located on a public mirror changes. This change of signature can cause issues for us because we download version tagged packages and still expect an older signature. Caching these packages locally may alleviate these issues.
```

Comment:
```suggestion
It is also possible that the [signature](https://wiki.archlinux.org/index.php/Pacman/Package_signing) on a package that is located on a public mirror changes. This change of signature can cause issues for us because we download version tagged packages and still expect an older signature. Caching such packages locally may alleviate these issues.
```